internal/pkg/pct/pct.go

```diff
@@ -154,6 +161,17 @@ func FormatTemplates(tmpls []PuppetContentTemplate, jsonOutput string) error {
 	return nil
 }
 
+// DisplayDefaults returns the Default values from a Templates configuration file
+func DisplayDefaults(defaults map[string]interface{}) string {
```

Comment:
This needs to return both formatted strings to the console as well as json formatted strings. The intention is for a tool like VSCode to call this and receive machine readable data.